umm-spec-lib/src/cmr/umm_spec/additional_attribute.clj

```diff
@@ -19,33 +19,33 @@
   (when data-type
     (csk/->SCREAMING_SNAKE_CASE_STRING (name data-type))))
 
-(defmulti parse-value
-  "Parses a value based on the data type given"
-  (fn [data-type value]
-    data-type))
-
-(defmethod parse-value :default
-  [data-type value]
-  (when value
-    (str value)))
-
-(defmethod parse-value "INT"
-  [data-type ^String value]
-  (when value (Long. value)))
-
-(defmethod parse-value "FLOAT"
-  [data-type ^String value]
-  (when value (Double. value)))
-
-(defmethod parse-value "BOOLEAN"
-  [data-type ^String value]
-  (when value
-    (case value
-      "true" true
-      "false" false
-      "1" true
-      "0" false
-      :else (errors/internal-error! (format "Unexpected boolean value [%s]" value)))))
+;(defmulti parse-value
+;  "Parses a value based on the data type given"
+;  (fn [data-type value]
+;    data-type))
+
+;(defmethod parse-value :default
+;  [data-type value]
+;  (when value
+;    (str value)))
+
+;(defmethod parse-value "INT"
+;  [data-type ^String value]
+;  (when value (Long. value)))
+
+;(defmethod parse-value "FLOAT"
+;  [data-type ^String value]
+;  (when value (Double. value)))
+
+;(defmethod parse-value "BOOLEAN"
+;  [data-type ^String value]
+;  (when value
+;    (case value
+;      "true" true
+;      "false" false
+;      "1" true
+;      "0" false
+;      :else (errors/internal-error! (format "Unexpected boolean value [%s]" value)))))
```

Comment:
The clojure style guide would recommend using `#_` over the literal comment tags to disable values.

Reply:
Thanks Jay, I was just making sure that what I changed won't break anything.  All of the functions that are commented out are all going to be deleted here in a few minutes.